File: src/org/ggp/base/player/request/factory/exceptions/RequestFormatException.java
package org.ggp.base.player.request.factory.exceptions;

class RequestFormatException(Exception):
{
    source = ''
    bad = Exception()

    def RequestFormatException(source='', Exception bad)
	{
        this.source = source;
        this.bad = bad;

    def String getSource()
	{
        return source;

    def String toString()
	{
        return "Improperly formatted request: " + source + ", resulting in exception: " + bad;

File: ggpy/cruft/autocode/RequestFormatException.py
#!/usr/bin/env python
""" generated source for module RequestFormatException """
# package: org.ggp.base.player.request.factory.exceptions
@SuppressWarnings("serial")
class RequestFormatException(Exception):
    """ generated source for class RequestFormatException """
    source = str()
    bad = Exception()

    def __init__(self, source, bad):
        """ generated source for method __init__ """
        super(RequestFormatException, self).__init__()
        self.source = source
        self.bad = bad

    def getSource(self):
        """ generated source for method getSource """
        return self.source

    def __str__(self):
        """ generated source for method toString """
        return "Improperly formatted request: " + self.source + ", resulting in exception: " + self.bad

